homeassistant/components/notify/notifyme.py

```diff
@@ -0,0 +1,52 @@
+"""
+NotifyMe platform for notify component.
+
+NotifyMe is an Alexa Skill which allows you to send any message to your
+Amazon Echo devices via a simple REST API.
+
+You have to install and set up the skill first to acquire an access code
+before you can set up this platform. See http://www.thomptronics.com/notify-me
+for complete instructions.
+"""
```

Comment:
Content like this should be moved to the documentation. A file header like all other platforms are using is enough.

Reply:
I have moved the doc. Pending reviews. https://github.com/home-assistant/home-assistant.io/pull/7650